apps/ledger-live-mobile/src/components/Nft/NftListItem.tsx

```diff
@@ -15,18 +15,16 @@ import {
 import { NFTResource } from "@ledgerhq/live-common/nft/NftMetadataProvider/types";
 import { Box, Flex, Tag, Text } from "@ledgerhq/native-ui";
 
-import { useNavigation } from "@react-navigation/native";
 import styled, { BaseStyledProps } from "@ledgerhq/native-ui/components/styled";
 import { useTranslation } from "react-i18next";
 import { getCryptoCurrencyById } from "@ledgerhq/live-common/currencies/index";
 import CurrencyIcon from "../CurrencyIcon";
 import NftMedia from "./NftMedia";
 import Skeleton from "../Skeleton";
-import { NavigatorName, ScreenName } from "../../const";
-import { track } from "../../analytics";
 
 type Props = {
   nft: ProtoNFT;
+  onPress?: (nft: ProtoNFT, metadata?: NFTMetadata) => void;
```

Comment:
metadata, rename to nftMetadata, could be better to to avoid confusion with NftCollectionMetadata

Reply:
Done